### DISCUSSION TITLE
Where to get approle role secret id and token

### DISCUSSION THREAD

**Discussion by srekkas** [bank-vaults/vault-operator#572] [Q&A] [answered]
Got vault installed, approles is created. But do role secret id and token is saved in some secret?

**Comment by csatib02** [comment, marked as answer]
Hey @srekkas,

They are saved in a k8s secret called: `vault-unseal-keys`.
If you are interested of retrieving the role:
```bash
vault auth list # To get enabled auth-methods

vault list auth/{auth-method}/role # To get associated role
```
